backend/wargamebackend/wargamelogic/consumers.py

```
@@ -282,15 +304,14 @@ async def handle_points_spend(self, data):
         sender_team_name = data["team_name"]
         sender_role_name = data["role_name"]
 
-        target_group = f"game_{self.join_code}_channel_{sender_team_name}{sender_role_name}_{sender_team_name}{sender_role_name}".replace(" ", "")
+        target_group = self.team_role_group
 
         if target_group not in self.channel_groups:
             username = self.scope["user"].username
             raise Exception(f"User {username} on team {sender_team_name} with role {sender_role_name} is not in group {target_group}")
```

Ruff (0.12.2)

Create your own exception

(TRY002)